src/inc/msquic.h

```diff
@@ -448,6 +448,7 @@ typedef struct QUIC_STATISTICS_V2 {
     uint32_t StatelessRetry         : 1;
     uint32_t ResumptionAttempted    : 1;
     uint32_t ResumptionSucceeded    : 1;
+    uint32_t GreaseBitNegotiated    : 1;    // Set if we negotiated the GREASE bit.
```

Comment:
Can you add a `uint32_t RESERVED : 27` too, so it's clear how much padding is here?

Reply:
Yes!